doc/introduction/framework/core.rst

```diff
@@ -0,0 +1,88 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _introduction/framework/core:
+
+*******************
+Core Pipeline layer
+*******************
+
+.. highlight:: bash
+
+Repository: https://github.com/ftrackhq/ftrack-connect-pipeline.git
+
+The core pipeline Framework module is the backbone of the pipeline, on which all
+other modules rely.
+
+It is in the core were all the interaction with the underlaying :term:`host type` is
+performed, except when it comes to bootstrap of the DCC.
+
+The core is depending on the :term:`definition` module to be present.
+
+The module composes four major components:
+
+ * The Host
```

Comment:
@henriknorin-ftrack 

link these entries to the page subsections so they become clickable